lib/buffer.js

```diff
@@ -637,6 +637,28 @@ Buffer.prototype.includes = function includes(val, byteOffset, encoding) {
   return this.indexOf(val, byteOffset, encoding) !== -1;
 };
 
+// Limit currently not implemented; reserved
+Buffer.prototype.split = function split(sep, limit) {
+  if (!limit)
+    limit = null;
+  if (!sep)
+    return [this];
+  if (!Buffer.isBuffer(sep))
+    throw new TypeError('First argument must be a buffer.');
```

Comment:
This is useless. Always use `Buffer::HasInstance()` in C++ and throw from there. The proto here can be screwed with.